dot/sync/disjoint_block_set_test.go

```diff
@@ -195,3 +196,24 @@ func TestDisjointBlockSet_getReadyDescendants_blockNotComplete(t *testing.T) {
 	require.Equal(t, block1.ToBlockData(), ready[0])
 	require.Equal(t, block2.ToBlockData(), ready[1])
 }
+
+func TestDisjointBlockSet_ClearBlocks(t *testing.T) {
+	s := newDisjointBlockSet(pendingBlocksLimit)
+
+	testHashA := common.Hash{0}
+	testHashB := common.Hash{1}
+
+	s.blocks[testHashA] = &pendingBlock{
+		hash:    testHashA,
+		clearAt: time.Now().Add(ttl * -2),
```

Comment:
nit you could use a common `now := time.Unix(1000, 0)` for both A and B so we don't rely on the clock. I'm a bit paranoid we usage of `time.Now()` in tests although it's probably fine here I think.

Reply:
updated